cumulus/components/userdata/windows.py

```diff
@@ -0,0 +1,33 @@
+from troposphere import (
+    Ref,
+    Join,
+)
+
+
+def user_data_for_cfn_init(launch_config_name, asg_name, configsets):
+    """
+    :return: A troposphere Join object that contains userdata for use with cfn-init
+    :param configsets: The single 'key' value set in the cfn-init Metadata parameter: cloudformation.InitConfigSets
+    :type asg_name: String name of the ASG cloudformation resource
+    :type launch_config_name: String name of the launch config cloudformation resource
+    """
+    default_userdata_asg_signal = (
+        Join('',
+             [
+                 "<powershell>\n",
+                 "& ", "$env:ProgramFiles\Amazon\cfn-bootstrap\cfn-init.exe",
+                 "         --stack ", Ref("AWS::StackName"),
+                 "         --resource ", launch_config_name,
+                 "         --configsets %s " % configsets,
+                 "         --region ", Ref("AWS::Region"), "\n",
+                 "# Signal the ASG we are ready\n",
+                 "&", "$env:ProgramFiles\Amazon\cfn-signal",
```

Comment:
This path needs to be fixed slightly. The cfn-signal is at `$env:ProgramFiles\Amazon\cfn-bootstrap\cfn-signal`

Reply:
@john-shaskin  this wasn't tested in the examples in this PR.  As discussed we'll merge this in and can be adjusted when we put this code through it's paces a little more.